Jenkinsfile

```diff
@@ -184,6 +184,41 @@ Map update_default_commit_pragmas() {
     }
 }
 
+Boolean skip_pragma_set(String name, String def_val='false') {
+    // Return whether or not the skip pragma is set
+    return cachedCommitPragma("Skip-${name}", def_val).toLowerCase() == 'true'
+}
+
+Boolean skip_build_stage(String distro='', String compiler='gcc') {
+    // Skip the stage if the CI_<distro>_NOBUILD parameter is set
+    if (distro) {
```

Comment:
`Boolean skip_build_stage(String distro, String compiler='gcc') {`
Do we have any use case to use this function with empty `distro`?